system-addon/content-src/lib/link-menu-options.js

```diff
@@ -120,6 +120,14 @@ export const LinkMenuOptions = {
     }),
     userEvent: "SAVE_TO_POCKET"
   }),
+  RemoveFromPocket: (site, eventSource) => ({
+    id: "menu_action_remove_pocket",
+    icon: "remove-pocket",
+    action: ac.AlsoToMain({
+      type: at.REMOVE_FROM_POCKET,
+      data: {pocket_id: site.pocket_id, url: site.url}
```

Comment:
If we make the reducer remove by `pocket_id` then we won't need `url` here anymore.